scrapy/commands/version.py

```diff
@@ -31,6 +32,7 @@ def run(self, args, opts):
             print("libxml2 : %s" % libxml2_version)
             print("Twisted : %s" % twisted.version.short())
             print("Python  : %s" % sys.version.replace("\n", "- "))
+            print("OpenSSL : %s" % OpenSSL.version.__version__)
```

Comment:
Is it a version of OpenSSL Python library or a version of OpenSSL system library?